src/app/find-help/view-offer-accommodation/view-offer-accommodation.component.ts

```diff
@@ -34,6 +36,7 @@ export class ViewOfferAccommodationComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    this.viewportScroller.scrollToPosition([0, 0]);
```

Comment:
I get the intent and it solves the issue but it's quite annoying when you scroll to the bottom of the offer and refresh the page and it scrolls back to the top. Ideally we'd only scroll when navigating via the router, not during page refresh. Is there a way to achieve that?

Reply:
Ok, see how it works now